jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListPresenter.java

```diff
@@ -406,4 +409,74 @@ public void setTaskService(final Caller<TaskService> taskService) {
+        );
+
+        view.addTextFilter(constants.Task(),
+                           constants.FilterByTaskName(),
+                           v -> addAdvancedSearchFilter(equalsTo(COLUMN_NAME,
+                                                                 v)),
+                           v -> removeAdvancedSearchFilter(equalsTo(COLUMN_NAME,
+                                                                    v))
+        );
+
+        final Map<String, String> status = getStatusByType(TaskType.ALL).stream().sorted().collect(Collectors.toMap(Function.identity(),
+                                                                                                                    Function.identity()));
+        view.addSelectFilter(constants.Status(),
+                             status,
+                             false,
+                             v -> addAdvancedSearchFilter(equalsTo(COLUMN_STATUS,
+                                                                   v)),
+                             v -> removeAdvancedSearchFilter(equalsTo(COLUMN_STATUS,
+                                                                      v))
+        );
+
+        view.addTextFilter(constants.Process_Instance_Correlation_Key(),
+                           constants.FilterByCorrelationKey(),
+                           v -> addAdvancedSearchFilter(equalsTo(COLUMN_PROCESS_INSTANCE_CORRELATION_KEY,
+                                                                 v)),
+                           v -> removeAdvancedSearchFilter(equalsTo(COLUMN_PROCESS_INSTANCE_CORRELATION_KEY,
+                                                                    v))
+        );
+
+        view.addTextFilter(constants.Actual_Owner(),
+                           constants.FilterByActualOwner(),
+                           v -> addAdvancedSearchFilter(equalsTo(COLUMN_ACTUAL_OWNER,
+                                                                 v)),
+                           v -> removeAdvancedSearchFilter(equalsTo(COLUMN_ACTUAL_OWNER,
+                                                                    v))
+        );
+
+        view.addTextFilter(constants.Process_Instance_Description(),
+                           constants.FilterByProcessInstanceDescription(),
+                           v -> addAdvancedSearchFilter(equalsTo(COLUMN_PROCESS_INSTANCE_DESCRIPTION,
+                                                                 v)),
+                           v -> removeAdvancedSearchFilter(equalsTo(COLUMN_PROCESS_INSTANCE_DESCRIPTION,
+                                                                    v))
+        );
+
+        //TODO missing creation date
+
+        final FilterSettings settings = view.getAdvancedSearchFilterSettings();
+        final DataSetFilter filter = new DataSetFilter();
+        filter.addFilterColumn(equalsTo(COLUMN_STATUS,
+                                        TASK_STATUS_READY));
```

Comment:
@cristianonicolai I think the default initial filter would be like the 'active' ones.. including :Ready, Reserved and InProgress status, do it make sense?